packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/openapi/openapi-validation.test.ts

```diff
@@ -135,6 +135,7 @@ test(testCase, async (t: Test) => {
   test(`${testCase} - ${fDeploy} - ${cOk}`, async (t2: Test) => {
     const parameters = {
       contractName: HelloWorldContractJson.contractName,
+      contractJSON: HelloWorldContractJson,
```

Comment:
@travis-payne Why is it necessary here to pass in the contractJSON? Double checking because want to make sure we maintain coverage and this test looks like it wants you to use the keychain instead?

Reply:
Removing this and adding a separate test case for deploying with args.